.ci_scripts/update_docs

```diff
@@ -17,7 +17,14 @@ if [ -n "$GH_TOKEN" ]; then
     if (( "$TRAVIS_BRANCH" == "master" && "$TRAVIS_PULL_REQUEST" == "false" )); then
         git commit -am "Re-ran make.py. [ci skip]" | true
         git remote add pushable https://${GH_TOKEN}@github.com/conda-forge/conda-forge.github.io.git/ > /dev/null
-        git push pushable new_site_content:master 2> /dev/null
+
+        # Update the remotes before pushing and check the status.
+        # This should give us more info if the push will not be
+        # a simple fast-forward push.
+        git branch -u pushable/master
```

Comment:
I'm just not familiar with the `-u` I guess - I have no problem with it in its current state if you have done some basic checks to confirm its behaviour.